src/BenchmarksDriver/Program.cs

```diff
@@ -23,8 +23,10 @@ public class Program
         private static readonly Dictionary<Scenario, ClientJob> _clientJobs =
             new Dictionary<Scenario, ClientJob>()
             {
-                { Scenario.Plaintext, new ClientJob() { Connections = 256, Threads = 32, Duration = 10, PipelineDepth = 16} },
-                { Scenario.Json, new ClientJob() { Connections = 256, Threads = 32, Duration = 10} },
+                { Scenario.Plaintext, new ClientJob() { Connections = 256, Threads = 32, Duration = 10, PipelineDepth = 16 } },
+                { Scenario.Json, new ClientJob() { Connections = 256, Threads = 32, Duration = 10 } },
+                { Scenario.MvcPlaintext, new ClientJob() { Connections = 256, Threads = 32, Duration = 10, PipelineDepth = 16 } },
+                { Scenario.MvcJson, new ClientJob() { Connections = 256, Threads = 32, Duration = 10} },
```

Comment:
I haven't done any private testing yet with MVC JSON, but the settings for MVC plaintext are (in my experience) the best.